pkg/naming/naming.go

```diff
@@ -3,14 +3,18 @@ package naming
 const (
 	perconaPrefix      = "percona.com/"
 	mysqlPerconaPrefix = "mysql.percona.com/"
+
+	kubernetesPrefix = "app.kubernetes.io/"
 )
 
 const (
-	LabelName      = "app.kubernetes.io/name"
-	LabelInstance  = "app.kubernetes.io/instance"
-	LabelManagedBy = "app.kubernetes.io/managed-by"
-	LabelPartOf    = "app.kubernetes.io/part-of"
-	LabelComponent = "app.kubernetes.io/component"
+	LabelName      = kubernetesPrefix + "name"
+	LabelInstance  = kubernetesPrefix + "instance"
+	LabelManagedBy = kubernetesPrefix + "managed-by"
+	LabelPartOf    = kubernetesPrefix + "part-of"
+	LabelComponent = kubernetesPrefix + "component"
+
+	LabelOperatorVersion = kubernetesPrefix + "version"
```

Comment:
I think that this const is only used in unit tests, maybe because we are adding the version to the crd directory through the generate command. If we don't use it actually somewhere else other than tests maybe it is better to move it in the test file and make it non-exported, or remove it completely and have a constant inside the test function and keep it scoped.